getting-started/gcp/main.tf

```diff
@@ -1,55 +1,72 @@
 provider "google" {
-  project = "terraform-project"
-  region  = "us-central1"
-  zone    = "us-central1-c"
+  credentials = file("<NAME>.json")
+
+  project = var.project
+  region  = var.region
+  zone    = var.zone
+}
+
+module "network" {
+  source  = "terraform-google-modules/network/google"
+  version = "1.1.0"
+
+  network_name = "terraform-vpc-network"
+  project_id   = var.project
+
+  subnets = [
+    {
+      subnet_name   = "subnet-01"
+      subnet_ip     = var.cidrs[0]
+      subnet_region = var.region
+    },
+    {
+      subnet_name   = "subnet-02"
+      subnet_ip     = var.cidrs[1]
+      subnet_region = var.region
+
+      subnet_private_access = "true"
+    },
+  ]
+
+  secondary_ranges = {
+    subnet-01 = []
+    subnet-02 = []
+  }
+}
+
+# Remove or #comment out:
+#
+# resource "google_compute_network" "vpc_network" {
+#   name = "terraform-network"
+# }
+
+resource "google_compute_address" "vm_static_ip" {
+  name = "terraform-static-ip"
 }
 
 resource "google_compute_instance" "vm_instance" {
   name         = "terraform-instance"
-  machine_type = "f1-micro"
+  machine_type = var.machine_types[var.environment]
+  tags         = ["web", "dev"]
 
-  labels = {
-    billing_department = "education"
+  provisioner "local-exec" {
+    command = "echo ${google_compute_instance.vm_instance.name}:  ${google_compute_instance.vm_instance.network_interface[0].access_config[0].nat_ip} >> ip_address.txt"
   }
 
   boot_disk {
     initialize_params {
-      image = "debian-cloud/debian-9"
+      image = "cos-cloud/cos-stable"
     }
   }
 
   network_interface {
-    network = google_compute_network.vpc_network.self_link
+# Replace this line:
```

Comment:
What does this comment do if the instructions are followed? Is this a task for the student or leftover from an 0.12 conversion script?

Reply:
Same as previous comment ^^